optking/tests/test_methimazole.py

```diff
@@ -21,10 +23,10 @@ def test_methimazole():
       H            0.135363403154     2.766598490457    -0.892748322003 
       H            0.687263403154    -2.310401509543     0.000151677997 
       H            2.917763403154    -1.162001509543     0.000551677997 
-    """)
-    psi4_options = {"basis" : 'pcseg-0'}
+    """
+    )
+    psi4_options = {"basis": "pcseg-0"}
     psi4.set_options(psi4_options)
-    result = optking.optimize_psi4('wb97x-d')
-    E = result['energies'][-1]
+    result = optking.optimize_psi4("wb97x-d")
```

Comment:
@AlexHeide , I'll start using " more often and spaces after my #'s so we don't get so many auto-edits!